crates/matrix-sdk-base/src/room/room_info.rs

```diff
@@ -480,7 +480,7 @@ impl RoomInfo {
             sync_info: SyncInfo::NoState,
             encryption_state_synced: false,
             latest_event: None,
-            read_receipts: Default::default(),
+            read_receipts: RoomReadReceipts::default(),
```

Comment:
Looks like all the changes in this file can now be reverted.

Reply:
Yeah, this is more about discovery and ease of reading outside an IDE. We have 500+ `Default::default`s throughout the SDK though so I guess I might as well get with the program.